NLog.Etw/NLog.Etw.csproj

```diff
@@ -51,18 +51,18 @@ Updated to latest NLog version
     <Reference Include="System.Core" />
     <Reference Include="Microsoft.CSharp" />
     <PackageReference Include="Microsoft.Diagnostics.Tracing.EventSource.Redist" Version="1.1.28" />
-    <PackageReference Include="NLog" Version="4.6.7" />
+    <PackageReference Include="NLog" Version="5.0.2" />
```

Comment:
Make make it a general dependency, instead of having it 3 times

Reply:
good one, fixed!